salt/cni/kube-flannel.yaml.jinja

```diff
@@ -47,6 +47,9 @@ data:
       "Backend":
       {
         "Type": "{{ salt.caasp_pillar.get('flannel:backend') }}"
```

Comment:
Trailing comma is needed as this is JSON. And more annoyingly, a trailing comma is forbidden if there is isn't another entry.

I'd move the Port entry to the top, add the trailing comma there, and it'll work

Reply:
You are right, thank you